shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -1443,6 +1443,26 @@ fn test_cassandra_peers_rewrite() {
     }
 }
 
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_cassandra_rewrite_peers_example() {
+    let _docker_compose =
+        DockerCompose::new("example-configs/cassandra-rewrite-peers/docker-compose.yml");
+
+    let shotover_manager = ShotoverManager::from_topology_file(
+        "example-configs/cassandra-rewrite-peers/topology.yaml",
+    );
+
+    let connection = shotover_manager.cassandra_connection("172.16.1.2", 9043);
+    table::test(&connection); // run some basic tests to confirm it works as normal
```

Comment:
We dont actually want to run any tests on it.
The image is old shotover code so who knows what features its missing.

Just starting up and shutting down the container should be enough to ensure that the configuration remains valid.

Also given that, I think this should go in its own file.
We definitely used to have a file for these sorts of "test config loads, but dont actually test functionality" cases, but I think we deleted that file due to all the tests being removed due to those examples being deleted.